fire/cli/info.py

```diff
@@ -372,40 +370,15 @@ def punkt(ident: str, obs: str, koord: str, detaljeret: bool, **kwargs) -> None:
         ident = f"{herred}-{sogn:02}-{lbnr:05}"
 
     try:
-        # Hvis ident er en uuid kan vi gå direkte på hegnspælen med "hent_punkt"...
-        uuidmønster = re.compile("[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*$")
-        if uuidmønster.match(ident):
-            pkt = firedb.hent_punkt(ident)
-            # TODO: Undersøg hvorfor det er nødvendigt at hejse flaget manuelt
-            if pkt is None:
-                raise NoResultFound
-            punkt_fuld_rapport(pkt, ident, 1, 1, obs, koord, detaljeret)
-            sys.exit(0)
-
-        # Ellers skal vi gennem en større søgesejlads...
-        punktinfo = (
-            firedb.session.query(pi)
-            .join(pit)
-            .filter(
-                pit.name.startswith("IDENT:"),
-                or_(
-                    pi.tekst == ident,
-                    pi.tekst.like(f"FO  %{ident}"),
-                    pi.tekst.like(f"GL  %{ident}"),
-                ),
-            )
-            .all()
-        )
-        if 0 == len(punktinfo):
-            raise NoResultFound
+        punkter = firedb.hent_punkter(ident)
     except NoResultFound:
         fire.cli.print(f"Error! {ident} not found!", fg="red", err=True)
         sys.exit(1)
 
     # Succesfuld søgning - vis hvad der blev fundet
-    n = len(punktinfo)
-    for i in range(n):
-        punkt_fuld_rapport(punktinfo[i].punkt, ident, i + 1, n, obs, koord, detaljeret)
+    n = len(punkter)
+    for i, punkt in enumerate(punkter):
+        punkt_fuld_rapport(punkt, punkt.ident, i + 1, n, obs, koord, detaljeret)
```

Comment:
Ja - det er vist den Pythoniske måde at gøre det på!

Reply:
Det er jo også svært at glemme 30+ års C erfaring :-)

Man burde nok også fjerne `punkt.ident` argumentet, da det jo er tilgængeligt på `punkt`. Det var bare ikke lige formålet med dette PR.

Reply:
> Det er jo også svært at glemme 30+ års C erfaring :-)

Godt nok er det ca. 34 år siden jeg første gang havde fingrene i en C-compiler (Lattice C 2.0), men rigtig brug af C kom jeg først i gang med i foråret 1990, i forbindelse med et kursus i remote sensing og billedbehandling på geografisk institut.

Ikke fordi det var strengt nødvendigt at programmere i C for at kunne udføre arbejdet til kurset, men fordi jeg havde opstøvet en slidt fotokopi af "Kernighan & Ritchie", som jo er en enestående pædagogisk bog, der gjorde det usædvanligt let at skrive C.

I øvrigt i den forbindelse jeg første gang mødte Niels Broge: Vi lavede kursusafsluttende projekt sammen. Den gang var det Borland C 2.0 - kort tid efter kom 3.0 med indbygget debugger og meget andet lækkert lakrids.

Jeg har først lært `enumerate()` at kende for nylig og synes det er en meget elegant, omend ikke strengt nødvendig, måde at gøre tingene på.